app/assets/javascripts/discourse/app/components/d-modal.gjs

```diff
@@ -121,39 +121,36 @@ export default class DModal extends Component {
   }
 
   @action
-  async handleSwipe(state) {
-    if (!this.site.mobileView) {
```

Comment:
this is handled at the modifier level with à@enabled={{this.site.mobileView}}`

Reply:
Would it make sense to move the mobileView check into `swipe` modifier itself? (i.e. are there cases where we want swipe enabled on desktop?)